packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -1,8 +1,10 @@
+/* eslint-disable max-lines */
 import { existsSync, mkdirSync, writeFileSync } from 'fs';
 import { resolve } from 'path';
-import { getBaseType, getFieldName, getSubscriptionName, GraphbackCoreMetadata, GraphbackOperationType, GraphbackPlugin, ModelDefinition, getTableName, getInputTypeName } from '@graphback/core'
+import { getFieldName, getSubscriptionName, GraphbackCoreMetadata, GraphbackOperationType, GraphbackPlugin, ModelDefinition, getInputTypeName, buildGeneratedRelationshipsFieldObject, getInputFieldName, canInputField, getInputFieldType, buildModifiedRelationshipsFieldObject } from '@graphback/core'
```

Comment:
This crazy big amount of helpers makes me worried. From someone taking a lot on those it will be realy hard to get them around. Plus we could have them namespaces as it is not clear what they involved. I will do wrap this into namespace and create follow up issue to create wrapper class that will give us them. There is no need to have them in core. We can simply have relationship object that offer those helpers

Reply:
Fully agree, I was thinking this myself

Reply:
100% agree with this.